### PR TITLE
feature: configurable deployment wait time

### PR DESCRIPTION
**Pull request by peter-svensson**

### CODE REVIEW

File: cmd/deploy/deploy.go [hunk 1]
```diff
@@ -31,10 +31,11 @@ func main() {
 }
 
 func doDeploy() int {
-	var context, namespace string
+	var context, namespace, timeout string
 	const (
 		contextUsage   = "override the context for default environment deployment target"
 		namespaceUsage = "override the namespace for default environment deployment target"
+		timeoutUsage   = "override the default deployment timeout (2 minutes)"
```
Comment (argoyle) [inline cmd/deploy/deploy.go]: Should we refer to the format allowed?

**Review by argoyle — APPROVED** [review]
Just a small comment. LGTM otherwise.

File: cmd/deploy/deploy.go (hunk 1 above)
Comment (peter-svensson) [reply cmd/deploy/deploy.go]: Definitely!

**Review by argoyle — APPROVED** [review]
🚀

**Pull request merged**